### DISCUSSION TITLE
Welcome to scalasql Discussions!

### DISCUSSION THREAD

**Discussion by lihaoyi** [com-lihaoyi/scalasql#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by caicancai** [comment]
Hello
I have a question, what is the difference between this and java jdbc? Have you considered doing some simple sql optimization in scalasql?

**Reply by minjibir** [reply]
JDBC is blocking from the root. Though, I have not yet, gone through this lib I would say it has better approach which is different than JDBC.
